src/main/java/misc/Ui.java

```diff
@@ -0,0 +1,159 @@
+package misc;
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.Objects;
+
+import exceptions.MissingParamsException;
+import exceptions.PositionException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Tasklist;
+import task.Todo;
+
+public class Ui {
+    String blankline = "____________________________________________________________ \s";
+    
+    /**
+     * Prints a farewell message to the user.
+     */
+    public void endGame() {
+        System.out.println(blankline + "\n" + "Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * Prints a start message.
+     */
+    public void startGame() {
+        System.out.println(  """
+            ____________________________________________________________ \s
+            Hello! I'm Bob \s
+            What can I do for you? \s
+            ____________________________________________________________ \s
+            """);
+    }
+
+    /**
+     * Return a blank line.
+     */
+    public String retBlank() {
+        return blankline;
+    }
```

Comment:
Perhaps the `retBlank()` method can be renamed to `getBlankline()` for more clarity?